### DISCUSSION TITLE
resolution of displacement series grid

### DISCUSSION THREAD

**Discussion by hardreddata** [insarlab/MintPy#504] [Q&A] [answered]
Hi,

Thanks for the ongoing support of this tool. 

I see Sentinel has IW pixel resolution of 20 x 22m and I have heard people talk about achieving this with InSAR techniques. 

My `save_qgis` output has points on a roughly 70 x 50m grid.

Can this resolution be tuned via parameters into either stackSentinel or MintPy? 

Many thanks.

**Comment by yunjunz** [comment, marked as answer]
Yes, they can be changed via command-line options of stackSentinel.py and template options (in the `11.1 geocode section`) in smallbaselineApp.py.
